src/Service/Program.cs

```diff
@@ -27,6 +27,13 @@ public class Program
 
         public static void Main(string[] args)
         {
+            // Compare current version of DAB with latest (non-rc) version in NuGet. 
+            VersionChecker.GetVersions(out string? latestVersion, out string? currentVersion);
```

Comment:
Should this check instead go to the CLI dab `start` command? instead of inside the engine?  That way docker isnt affected if this wasnt meant for that scenario..

Reply:
Doing this version check everytime at the start of the engine seems time consuming, moving to dab init ensure the developers get this message when they initialize the configuration.

Reply:
I moved it to only run in `validate` which seems like a slow introduction.